src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -130,6 +133,28 @@
           <ButtonText>&amp;Package Manager Settings</ButtonText>
         </Strings>
       </Button>
+
+      <Button guid="guidDialogCmdSet" id="cmdidUpdatePackage" priority="0x0100" type="Button">
+        <Parent guid="guidPackageManagementCmdSet" id="idPackageUpdateGroup" />
+        <CommandFlag>AllowParams</CommandFlag>
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <CommandFlag>DynamicVisibility</CommandFlag>
+        <Strings>
+          <CommandName>cmdidUpdatePackage</CommandName>
+          <ButtonText>U&amp;pdate...</ButtonText>
+        </Strings>
+      </Button>
+
+      <Button guid="guidDialogCmdSet" id="cmdidUpdatePackages" priority="0x0100" type="Button">
+        <Parent guid="guidPackageManagementCmdSet" id="idPackageManagementGroup" />
+        <CommandFlag>AllowParams</CommandFlag>
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <CommandFlag>DynamicVisibility</CommandFlag>
+        <Strings>
+          <CommandName>cmdidUpdatePackages</CommandName>
+          <ButtonText>U&amp;pdate...</ButtonText>
```

Comment:
Should the text say "Update all..."?

Reply:
I thought so too but the original spec indicates that it should read "Update..." for both the individual packages and the Packages node.